bucket/fysty.json

```
@@ -0,0 +1,52 @@
+{
+    "version": "20220202-76f785d5",
+    "description": "A fork of the SSH/Telnet client PuTTY with new features such as background images, transparent window, clickable URLs and zooming.",
+    "homepage": "https://github.com/lalbornoz/FySTY",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/lalbornoz/FySTY/releases/download/FySTY-Release-76f785d5/FySTY-Release-76f785d5.zip",
+            "hash": "0f87de7cb30320f7260a25f20e2139416dc9c2417bf35ee306ae135dd81c2f7b"
+        }
+    },
+    "extract_dir": "FySTY-Release-76f785d5",
+    "bin": [
+        "bidi_gettype.exe",
+        "bidi_test.exe",
+        "pageant.exe",
+        "plink.exe",
+        "pscp.exe",
+        "psftp.exe",
+        "psocks.exe",
+        "putty.exe",
+        "puttygen.exe",
+        "puttytel.exe"
+    ],
+    "shortcuts": [
+        [
+            "putty.exe",
+            "PuTTY"
+        ],
+        [
+            "puttygen.exe",
+            "PuttyGen (PuTTY Key Generator)"
+        ],
+        [
+            "pageant.exe",
+            "Pageant (PuTTY authentication agent)"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/lalbornoz/FySTY",
+        "regex": "(?sm)datetime=\"(\\d{4})-(\\d{2})-(\\d{2}).*?FySTY-Release-(?<commit>[0-9a-f]{8})\\.zip",
+        "replace": "${1}${2}${3}-${4}"
```

```suggestion
        "github": "https://api.github.com/repos/lalbornoz/FySTY",
        "regex": "(?s)Release-(?<sha>[0-9a-f]{8}).*?updated_at.*?(\\d{4})-(\\d{2})-(\\d{2})",
        "replace": "${1}${2}${3}-${sha}"
```

API downloads less data.